src/types.ts

```diff
@@ -126,6 +126,7 @@ export interface Task extends RaRecord {
     text: string;
     due_date: string;
     done_date?: string | null;
+    sales_id: Identifier;
```

Comment:
Should be `sales_id?: Identifier;` since the column does not have the `not null` constraint in DB?